DigitalLearningSolutions.Data/DataServices/LearningLogItemsDataService.cs

```diff
@@ -154,5 +192,23 @@ public void UpdateLearningLogItemLastAccessedDate(int id, DateTime lastAccessedD
                 new { id, lastAccessedDate }
             );
         }
+
+        public void SetCompletionDate(int learningLogItemId, DateTime? completedDate)
+        {
+            var numberOfAffectedRows = connection.Execute(
+                @"UPDATE LearningLogItems
+                        SET CompletedDate = @completedDate
+                        WHERE LearningLogItemID = @learningLogitemId",

```

Comment:
It won't affect anything, but could we make "@learningLogitemId" become "@learningLogItemId" (capitalisation)

Reply:
oops, changed!